lib/chrono_model/utils.rb

```diff
@@ -51,7 +51,8 @@ def amend_period!(hid, from, to)
 
       connection.execute %[
         UPDATE #{quoted_table_name}
-           SET "validity" = tsrange(#{connection.quote(from)}, #{connection.quote(to)})
+           SET "validity" = tsrange(#{connection.quote(from)}, #{connection.quote(to)}),
+               "recorded_at" = #{connection.quote(from)}
```

Comment:
@pzac Thanks for reviewing!
The purpose of `amend_period!` is to alter history entries (for example in my case I need to generate a fake project history throughout a year for testing purposes) so we want to set the validity to whatever `ts_range`, and `recorded_at` should match `from` in order to keep working the chronological order on history - see https://github.com/ifad/chronomodel/blob/master/lib/chrono_model/time_machine.rb#L57

/cc @vjt